api/turing/internal/ref/types.go

```diff
@@ -0,0 +1,19 @@
+package ref
```

Comment:
I took the inspiration from this [aws/aws-sdk-go](https://github.com/aws/aws-sdk-go/blob/9d10b7469ebfe21f9ab825461b034f5ac6fc4b8b/aws/convert_types.go#L64) and this package is to dynamically get a reference (address) of a const golang value. It's currently used in tests only, however it's not necessarily be the case in the future. Hence, I'd like to keep in `internal` instead of `testutils`